test/addons-napi/test_dataview/test_dataview.c

```diff
@@ -0,0 +1,48 @@
+#include <node_api.h>
+#include <string.h>
+#include "../common.h"
+
+napi_value CreateDataView(napi_env env, napi_callback_info info) {
+  size_t argc = 1;
+  napi_value args [1];
+  NAPI_CALL(env, napi_get_cb_info(env, info, &argc, args, NULL, NULL));
+
+  NAPI_ASSERT(env, argc == 1, "Wrong number of arguments");
+
+  napi_valuetype valuetype;
+  napi_value input_dataview = args[0];
+
+  NAPI_CALL(env, napi_typeof(env, input_dataview, &valuetype));
+  NAPI_ASSERT(env, valuetype == napi_object,
+              "Wrong type of argments. Expects a dataview  as the first"
```

Comment:
Spelling: `argments`, and an additional space between `dataview` and `as`.
As `dataview` refers to a type here, I think `DataView` would be better.